mltsp/Flask/flask_app.py

```diff
@@ -681,6 +659,86 @@ def list_featuresets(
         return authed_featuresets
 
 
+# TODO database module
+def list_datasets_cursor_to_html_table(cursor):
```

Comment:
Why do we operate directly on cursors?  Perhaps it is best to query the database, get the data into a format we can easily use, and then simply inject that data into the template.  Generating HTML in the library does not feel right.